inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/InAppMessaging.kt

```diff
@@ -74,6 +74,13 @@ abstract class InAppMessaging internal constructor() {
     @RestrictTo(RestrictTo.Scope.LIBRARY)
     internal abstract fun getHostAppContext(): Context?
 
+    /**
+     * This methods manually closes the currently displayed message.
```

Comment:
"This method manually" is superfluous. Just
>  Close the currently displayed message.

Reply:
Updated to `Close the currently displayed message.`